src/index.html

```diff
@@ -4,20 +4,19 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- Twitter -->
+    <meta name="twitter:card" content="summary" />
     <!-- OpenGraph -->
-    <meta property="og:title" content="ESA - Climate from space" />
     <meta property="og:type" content="website" />
+    <meta property="og:title" content="ESA - Climate from Space" />
     <meta
       property="og:image"
-      content="https://storage.googleapis.com/esa-cfs-storage/0.6.1/stories/a-passage-opens/assets/sst_large_18.jpg"
-    />
-    <meta
-      property="og:url"
-      content="https://storage.googleapis.com/esa-cfs-versions/web/master/index.html"
+      content="http://cfs.climate.esa.int/assets/images/share.png"
     />
+    <meta property="og:url" content="https://cfs.climate.esa.int" />
     <meta
       property="og:description"
-      content="Climate from Space application for ESAs CCI+ program."
+      content="Climate from Space web application for ESA’s CCI programme"
     />
     <title>ESA - Climate from space</title>
```

Comment:
maybe:
```suggestion
    <title>ESA - Climate from Space</title>
```
for more consistency

Reply:
👍